src/modules/m_filter.cpp

```diff
@@ -305,8 +312,12 @@ CmdResult CommandFilter::Handle(User* user, const Params& parameters)
 
 bool ModuleFilter::AppliesToMe(User* user, FilterResult* filter, int iflags)
 {
+	const AccountExtItem* accountext = GetAccountExtItem();
+
 	if ((filter->flag_no_opers) && user->IsOper())
 		return false;
+	if ((filter->flag_no_registered) && accountext && accountext->get(user))
```

Comment:
This line of code means:

>If the registered flag is set AND accounts exist AND the user is logged into an account

I think you probably meant to do `!accountext->get(user)`?